### DISCUSSION TITLE
请问一下生成静态文件放到服务器上之后怎么配置nginx哇

### DISCUSSION THREAD

**Discussion by jiankaitian** [imsyy/home#202] [Q&A]
配置成这样不行吗
server {
    listen 80;
    server_name example.com;

    location / {
        root /path/to/your/home/dist;
        try_files $uri $uri/ /index.html;
    }
}

**Comment by DogTang0323** [comment]
你这是啥 这不是模板吗

**Comment by SoiTeoh** [comment]
1、server_name example.com;
example.com这要改为自己网站的域名
2、root /path/to/your/home/dist;
把这个目录改成服务器上存放静态文件的位置

**Reply by SoiTeoh** [reply]
不好意思，现在才看到消息。可以不用域名，用ip，但必须要有一个域名，并且做好域名解析，只用ip是不行的。
